docs/Configuration.md

```diff
@@ -82,14 +84,14 @@ export default config;
 
 
 ## Gluestick config
-To modify GlueStick config use `src/gluestick.config.js` file. It must return function or array of functions
+To modify the GlueStick config use `src/gluestick.config.js`. It must return a function or array of functions
 as a `default` export:
 ```
 export default config => config;
 // or
 export default [config => config];
 ```
-This function accepts gluestick config (`object`) as a first argument and must return modified
+This function accepts the gluestick config (`object`) as a first argument and must return the modified
```

Comment:
I wonder if we should just substitute all uses of `gluestick` with `GlueStick` in this file (unless it's the command)